overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -45,6 +47,8 @@ type refreshCommand struct {
 	// these two options are mutually exclusive
 	Proceed bool `long:"proceed" description:"Proceed with potentially disruptive refreshes"`
 	Hold    bool `long:"hold" description:"Do not proceed with potentially disruptive refreshes"`
+
+	PrintInhibitLock bool `long:"show-inhibit-lock" description:"Show inhibit lock value (may be empty)"`
```

Comment:
this can probably be just --show-lock  as the context is refresh?

The description might be a bit longer:

Show the value of the run inhibit lock held during refreshes (empty means not held)

Reply:
Done.